ramls/rtypes/simple-crud.raml

```diff
@@ -0,0 +1,114 @@
+#%RAML 1.0 ResourceType
+# item-collection.raml
+    description: Entity representing a <<resourcePathName|!singularize>>
+    #is: [language]
+    post:
```

Comment:
Optional. The content of this raml resource type is almost the same as in `raml-util/rtypes/item-collection.raml` except `post` part. Probably `item-collection.raml` as a type for `/credentials` and post can be declared separately in `interface.raml`